bios/mrmorris.md

```diff
@@ -1,5 +1,9 @@
-Ryan Morris
+# Ryan Morris
 
 * A human
 * Not a dog :dog:
 * Has fingers :hand:
+
+## Favorite Vacation Spot
+
+* Singapore
```

Comment:
What if you included the country too?  Is there more than one Singapore?  I am terrible with geography! :see_no_evil: